src/comments.ts

```diff
@@ -18,7 +18,7 @@ export const HadError = (user: string | undefined, error: string) => ({
 
 export const CIFailed = (abbrOid: string, user: string, ciUrl: string) => ({
     tag: `gh-actions-complaint-${abbrOid}`,
-    status: `@${user} The CI build failed! Please [review the logs for more information](${ciUrl}).\r\n\r\nOnce you've pushed the fixes, the build will automatically re-run. Thanks!`
+    status: `@${user} The CI build failed! Please [review the logs for more information](${ciUrl}).\r\n\r\nOnce you've pushed the fixes, the build will automatically re-run. **Note: builds which are failing do not end up on list of PRs for DT maintainers to review.** Thanks!`
```

Comment:
Shouldn't there be an empty line before this so it's on its own line?

(Assuming that you're out for the day I'll just do that...)